lib/taurus/core/util/log.py

```diff
@@ -793,6 +793,15 @@ def _format_stack(self, stack_func=inspect.stack):
                         out += '<could not find suitable string representation>'
         return out
 
+    def loggingtofile(self, msg, logging_path, *args, **kw):
+
+        file_name = logging_path + "/spock_session.log"
+
+        logging_file = open(file_name, 'a')
+        logging_file.write('\n')
+        logging_file.write(msg)
+        logging_file.close()
+
```

Comment:
Now a more general comment. My knowledge of the `logging` module is a bit rusty, but I think that the feature that you are implementing with this `loggingtofile` method may already be covered in the `logging` module in a more generic form.
@cmft is the expert on this and may have something to say